.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -8,16 +8,10 @@ about: Suggest an improvement/change to the docs
 ## Describe the change
 <!-- A clear and concise description of what you want to happen)-->
 
-
-## How will this impact users?
-
-
-## User Research Evidence
-
-<!-- Does evidence exist from user research to support this idea? Could research or design help validate or improve it? Has this idea been influenced by your conversations with users, or by your own experience as a SecureDrop user? If so, please note it here, and [take a look at our UX wiki](https://github.com/freedomofpress/securedrop-ux/wiki) for more information about our design and research efforts.-->
-
+## How will this impact [SecureDrop users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?
```

Comment:
This repo also includes development documentation - developers are not referenced in the wiki page above. Maybe drop the link and say "How will this impact SecureDrop users or developers"?

Reply:
@zenmonkeykstop Now that the developer docs have been moved to their own repo, I'm thinking that the concern you flagged above no longer needs to be addressed here. Do you have any reservations about moving forward with merging this PR?